internal/models/pubkey_model.go

```diff
@@ -56,3 +58,9 @@ func (pk *Pubkey) FindAwsFingerprint(ctx context.Context) string {
 
 	}
 }
+
+func (pk *Pubkey) GetPubkeyWithUsername(ctx context.Context) string {
```

Comment:
```suggestion
func (pk *Pubkey) BodyWithUsername(ctx context.Context) string {
```

Maybe?

Reply:
To give you context: getters wit Get are not welcomed in Go.

https://go.dev/doc/effective_go#Getters